master.css

```diff
@@ -0,0 +1,112 @@
+
+.leftSide {
+    height: 90%;
+    background-color: rgb(0, 162, 255);
+    position: absolute;
+    left: 0%;
+    width: 50%;
+    background-image: url("./images/birdbackground.png");
+    background-repeat: no-repeat;
+    
+}   
+
+.rightSide {
+    height: 90%;
+    position: absolute;
+    left: 60%;
+    width: 70%;
+    top: 0%;
+    bottom: 0%;
+    background-image: url("./images/birdlogo.png");
+    background-repeat: no-repeat;
+    background-position: left 50%;
+    background-position: top 40%;
+    background-size: 150px;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+    
+    
+    /* background-position: ; */
+    /* align-self: auto; */
+}
+.rightCenter {
+    position: absolute;
+    left: 65%;
+    width: 70%;
+    bottom: 30%;
+    
+    
+}
+
+#foot {
+    flex-wrap: nowrap;
+    display: flex;
+    justify-content:space-evenly;
+    list-style:none;
+    position: absolute;
+    left: 0%;
+    bottom: 0%;
+    width: 90%;
+}
+
+#leftList {
+    text-align: center;
+    list-style: none;
+    position: absolute;
+    left: 25%;
+    bottom: 50%;
+    text-align: left;
+    color: white;
+    align-items: center;
+    
+}
+#leftList li {
+    padding-bottom: 10%;
+    position: relative;
+    bottom: 50%;
+    
+    /* align-items: center; */
+
+}
+/* #rightList {
+    justify-content: space-around;
+} */
+button{
+    
+    border: 2px solid rgb(64, 173, 224);
+    border-radius: 20px;
+    color: rgb(64, 173, 224);
+    /* width: 20%; */
+    margin: 12px 0px 0px;
+    padding: 12px;
+
+}
+.buttons {
+    border: 2px solid rgb(64, 173, 224);
+    border-radius: 20px;
+    color: rgb(64, 173, 224);
+    width: 30%;
+    padding: 10px 10px;
+    margin: 8px;
+    
+}
+button:hover {
```

Comment:
With this you can also change the color of the button when its being hovered over.